webapp/src/main/java/com/box/l10n/mojito/android/strings/AndroidStringDocumentMapper.java

```diff
@@ -165,6 +165,10 @@ boolean isSingularTextUnit(TextUnitDTO textUnit) {
         return Strings.isNullOrEmpty(textUnit.getPluralForm());
     }
 
+    String textUnitKey(TextUnitDTO textUnit){
+        return textUnit.getAssetPath() + "_" + textUnit.getPluralFormOther();
```

Comment:
don't we have an asset delimiter like `"#@#` ? for singular?

Reply:
Updated